packages/docs/fluent-editor/demos/flow-chart-resize.vue

```
@@ -0,0 +1,53 @@
+          deps: {
+            LogicFlow,
+            DndPanel,
+            SelectionSelect,
+            Snapshot,
+          },
+        },
+      },
+    })
+    const ops = [{ insert: '\n' }, { insert: { 'flow-chart': { nodes: [{ id: 'node1', type: 'rect', x: 100, y: 150, text: '开始' }, { id: 'node2', type: 'rect', x: 300, y: 150, text: '结束' }], edges: [{ id: 'edge1', sourceNodeId: 'node1', targetNodeId: 'node2', type: 'polyline' }] } } }, { insert: '\n\n' }]
+    editor.setContents(ops)
+  })
```

ESLint

[error] Should have line breaks between items, in node CallExpression

(antfu/consistent-list-newline)